docs/development.rst

```diff
@@ -92,3 +92,39 @@ This file configures the behavior of the a pre-commit hook based on `the Pre-Com
 installing it (with pip, brew or other tools) then run ``pre-commit install`` in the repo's root to activate the hook.
 For the full use of the tool, you may need to install also other third-party tools, such as
 `shellcheck <https://github.com/koalaman/shellcheck/>`_ and `shfmt <https://github.com/mvdan/sh>`_.
+
+Running pre-commit manually
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once configured, pre-commit will auto-run against staged files as part
+of the commit process.
+
+You can also run pre-commit manually to check staged files without
+having to initiate a commit:
+
+::
+
+    $ pre-commit
+
+shellcheck and $BASH\_IT variable
```

Comment:
I think this part is no longer needed after #1701 
It should instead have information about `lint_clean_files.sh`

Reply:
Heya,

I'm ok removing it, but I don't think it hurts to have it, either.

If the user doesn't perform a `BASH_IT='' pre-commit` check, the worst that happens is that the sourcing error gets kicked down the road until the first pipeline run where the file is also present in clean_files.txt

Related Question: is our plan to expect new PRs to add their changed/added files/folders to the clean_files.txt as *part* of the PR?
Actually I think I'll start an issue so we can discuss this.

-D

Reply:
If you do not want to remove it, at least recommend running `lint_clean_files.sh`
after adding their file

Reply:
@davidpfarrell what do you say about adding here a section about `clean_files.txt` and `lint_clean_files.sh` in order to help new contributors to know how to properly lint their files? We should tell them to always add new files to `clean_files.txt` and consider adding files if they only modify existing file